mlflow/paddle/__init__.py

```diff
@@ -446,3 +448,103 @@ def predict(self, data):
 def _contains_pdparams(path):
     file_list = os.listdir(path)
     return any(".pdparams" in file for file in file_list)
+
+
+@experimental
+@autologging_integration(FLAVOR_NAME)
+def autolog(
+    log_every_n_epoch=1, log_models=True, disable=False, exclusive=False, silent=False,
+):  # pylint: disable=unused-argument
+    """
+    Enables (or disables) and configures autologging from `PaddlePaddle
+    <https://pytorch-lightning.readthedocs.io/en/latest>`_ to MLflow.
+
+    Autologging is performed when you call the `fit` method of
+    `paddle.Model \
+    <https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/Model_en.html#>`_.
```

Comment:
Can we fix this line like below?

```suggestion
    Autologging is performed when you call the `fit` method of `paddle.Model`_.

    .. _paddle.Model:
        https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/Model_en.html
```

Reply:
Done